spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/secretmanager/GcpSecretManagerBootstrapConfiguration.java

```diff
@@ -62,16 +63,16 @@ public GcpSecretManagerBootstrapConfiguration(
 				? properties::getProjectId
 				: new DefaultGcpProjectIdProvider();
 
-		// Registers {@link ByteString} type converters to convert to String and byte[].
-		configurableEnvironment.getConversionService().addConverter(
```

Comment:
Can we just register in both places to make it work for 2.4 and 2.5?

Reply:
Yeah that would work. Following the thread I wonder if the Spring may know of a better solution.